js/Store.js

```diff
@@ -75,6 +81,8 @@ const store = {
     emitTo(componentName, event, ...params) {
         let components = this.getComponentsByName(componentName)
 
+        components = components.concat(this.getComponentsByKey(componentName))
```

Comment:
I can understand your code, but this lead possible to confusing later. I think a dedicated `$emitToKey` function would be better to seperate responsibilities?